client/my-sites/migrate/components/sites-block/index.jsx

```diff
@@ -39,7 +39,7 @@ class SitesBlock extends Component {
 								className="sites-block__faux-site-selector-label"
 								htmlFor="sites-block__faux-site-selector-url-input"
 							>
-								Import from...
+								{ translate( 'Import from…' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **107** times:
`translate( 'Import from JSON' )` *ES Score: 6*